Example/Example/Common/BankCard/BankCardViewController.swift

```diff
@@ -14,6 +14,50 @@
 
 import TextInputKit
 
+struct BankCard {
+
+    var cardNumber: BankCardNumber? = nil
```

Comment:
Optional vars are nil by default in Swift. Is there something wrong with structures initializer?